src/components/NewTodo.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC, memo } from 'react';
+
+interface Props {
+  newTodoField: React.RefObject<HTMLInputElement>,
+  onFocus: React.Dispatch<React.SetStateAction<boolean>>,
+}
+
+export const NewTodo: FC<Props> = memo(
+  ({ newTodoField, onFocus }) => (
+    <header className="todoapp__header">
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
If you don't want to see the error, you can use:
```
aria-label=[the text that describes the button]

Reply:
thx, useful information)